### DISCUSSION TITLE
Feature Request: Hide the icon from Dock and move to the statusbar

### DISCUSSION THREAD

**Discussion by tfactor2** [ghostty-org/ghostty#3946] [Feature Requests, Ideas] [answered]
Summary
=======

It would be great to add an option to hide the app from the Dock, just like it's done in iTerm2.
![image](https://github.com/user-attachments/assets/299dd22a-99ae-4987-ad5a-1e7830e89c7c)

Motivation / Use Case
========

The terminal is an "always-on" application, and its presence in the Dock can be distracting. It would be great to keep the Dock "lean."

**Comment by JakeWharton** [comment, marked as answer]
#3810

**Comment by tfactor2** [comment]
Indeed, thanks for the reference - can be closed as a duplicate of https://github.com/ghostty-org/ghostty/discussions/3810.
